Add mySQL index metric privileges (#28351)
DataDog/documentation: content/en/database_monitoring/setup_mysql/troubleshooting.md

Before following these steps to diagnose missing query metric data, ensure the Agent is running successfully and you have followed [the steps to diagnose missing agent data](#no-data-is-showing-after-configuring-database-monitoring). Below are possible causes for missing query metrics.

### Index metrics are missing

> The Agent displaying this error:

Reviewer: ```suggestion
If the Agent displays this error:
```

```
Error querying mysql.innodb_index_stats: (1142, "SELECT command denied to user 'datadog'@'172.20.0.5' for table 'innodb_index_stats'")
```

> Can be resolved by granting the `datadog` user the SELECT privilege to collect index metrics:

Reviewer: ```suggestion
Resolve the error by granting the `datadog` user the SELECT privilege to collect index metrics:
```
